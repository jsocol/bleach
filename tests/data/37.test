<div onclick=alert(0)><form onsubmit=alert(1)><input onfocus=alert(2) name=parentNode>123</form></div>
