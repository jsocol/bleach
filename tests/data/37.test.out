<div><form><input>123</form></div>
